style.css

```diff
@@ -93,3 +94,224 @@ body {
   margin: 0;
   padding: 0;
 }
+
+.works {
+  display: flex;
+  background-color: #fff;
+  padding: 16px;
+  flex-direction: column;
+}
```

Comment:
- I think you may have forgotten to separate each of your project cards. Currently, there are no borders diving one card from the next. Can we fix that??
- Also, I noticed that your mobile portfolio isn't displayed fully on the browser (screensize 375px). Some content of the project card is currently overflowing. I mean the text `2015` is outside of the mobile website. Can we fix that too?

Here is what I mean 👇🏽 

|  Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="401" alt="Screenshot 2023-01-20 at 15 13 40" src="https://user-images.githubusercontent.com/99246359/213807587-525f7b62-051f-487b-aa9c-5d2c623641c6.png"> |<img width="235" alt="Screenshot 2023-01-20 at 15 28 22" src="https://user-images.githubusercontent.com/99246359/213808539-af66c2a0-dad1-4416-9756-134eac4d3f91.png">  |

⚠️ I think it is best we stick to the design on Figma as much as possible
Kindly take note of ;
`The borders dividing each project card from the next`
`The extra space on the right, just like the image i added above`
`The overflowing text`